apollo-router/src/configuration/mod.rs

```diff
@@ -161,10 +161,6 @@ pub struct Configuration {
     #[serde(default)]
     pub(crate) experimental_chaos: Chaos,
 
-    /// Set the API schema generation implementation to use.
-    #[serde(default)]
-    pub(crate) experimental_api_schema_generation_mode: ApiSchemaMode,
```

Comment:
Yes, old configs using this will (I think) cause a “Additional properties are not allowed” error.

To avoid that please add:
* A migration file under `apollo-router/src/configuration/migrations/`
* A test config file under `apollo-router/src/configuration/testdata/migrations/`

Reply:
Thanks!